packages/server/graphql/mutations/signUpWithPassword.ts

```diff
@@ -41,6 +41,9 @@ const signUpWithPassword = {
   },
   resolve: rateLimit({perMinute: 50, perHour: 500})(
     async (_source: unknown, args: SignUpWithPasswordMutationVariables, context: GQLContext) => {
+      if (process.env.AUTH_INTERNAL_DISABLED === 'true') {
```

Comment:
+1 part of me wants to do this check in rootMutation to remove it from the schema entirely, but this is probably good